keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json

```
@@ -0,0 +1,167 @@
+                {"label": "KC_S", "matrix": [2, 2], "x": 2.75, "y": 2, "w": 1},
+                {"label": "KC_D", "matrix": [2, 3], "x": 3.75, "y": 2, "w": 1},
+                {"label": "KC_F", "matrix": [2, 4], "x": 4.75, "y": 2, "w": 1},
+                {"label": "KC_G", "matrix": [2, 5], "x": 5.75, "y": 2, "w": 1},
+                {"label": "KC_H", "matrix": [2, 6], "x": 6.75, "y": 2, "w": 1},
+                {"label": "KC_J", "matrix": [2, 7], "x": 7.75, "y": 2, "w": 1},
+                {"label": "KC_K", "matrix": [2, 8], "x": 8.75, "y": 2, "w": 1},
+                {"label": "KC_L", "matrix": [2, 9], "x": 9.75, "y": 2, "w": 1},
+                {"label": "KC_SCLN", "matrix": [2, 10], "x": 10.75, "y": 2, "w": 1},
+                {"label": "KC_QUOT", "matrix": [2, 11], "x": 11.75, "y": 2, "w": 1},
+                {"label": "KC_ENTER", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "KC_PGDN", "matrix": [2, 15], "x": 15, "y": 2, "w": 1},
+                {"label": "KC_LSFT", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
+                {"label": "KC_Z", "matrix": [3, 1], "x": 2.25, "y": 3, "w": 1},
+                {"label": "KC_X", "matrix": [3, 2], "x": 3.25, "y": 3, "w": 1},
+                {"label": "KC_C", "matrix": [3, 3], "x": 4.25, "y": 3, "w": 1},
+                {"label": "KC_V", "matrix": [3, 4], "x": 5.25, "y": 3, "w": 1},
+                {"label": "KC_B", "matrix": [3, 5], "x": 6.25, "y": 3, "w": 1},
+                {"label": "KC_N", "matrix": [3, 6], "x": 7.25, "y": 3, "w": 1},
+                {"label": "KC_M", "matrix": [3, 7], "x": 8.25, "y": 3, "w": 1},
+                {"label": "KC_COMM", "matrix": [3, 8], "x": 9.25, "y": 3, "w": 1},
+                {"label": "KC_DOT", "matrix": [3, 9], "x": 10.25, "y": 3, "w": 1},
+                {"label": "KC_SLSH", "matrix": [3, 10], "x": 11.25, "y": 3, "w": 1},
+                {"label": "KC_RSFT", "matrix": [3, 11], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "KC_UP", "matrix": [3, 14], "x": 14, "y": 3, "w": 1},
+                {"label": "KC_END", "matrix": [3, 15], "x": 15, "y": 3, "w": 1},
+                {"label": "KC_LCTL", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "KC_LGUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "KC_LALT", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "KC_SPC", "matrix": [4, 5], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "KC_RALT", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
+                {"label": "_FN", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "KC_LEFT", "matrix": [4, 13], "x": 13, "y": 4, "w": 1},
+                {"label": "KC_DOWN", "matrix": [4, 14], "x": 14, "y": 4, "w": 1},
+                {"label": "KC_RGHT", "matrix": [4, 15], "x": 15, "y": 4, "w": 1}
+            ]
+        }
+    }
```

move to info.json, as LAYOUT_65_ansi_blocker
```suggestion
        }
```